frontend/src/Components/AlgInputs/AlgInputsLoadControl.js

```diff
@@ -34,7 +34,10 @@ const styles = theme => ({
     },
 });
 
+<<<<<<< HEAD
```

Comment:
This git's merge conflict markers. Please fix the merge conflicts and remove them. The code won't work otherwise.

Reply:
my apologies. I merged some changes from master recently, and didn't catch this. Thanks.